pac4j-core/src/main/java/org/pac4j/core/matching/PathMatcher.java

```diff
@@ -89,6 +90,14 @@ boolean matches(final String path) {
         return excludedPatterns;
     }
 
+    public void setExcludedPaths(Collection<String> paths) {
+        paths.forEach(path -> excludePath(path));
```

Comment:
Wouldn't it be better if this method remove previous values? Since it is called set and not add... sorry :)